packages/create-react-native-library/templates/native-common/android/build.gradle

```diff
@@ -112,69 +152,24 @@ repositories {
   mavenCentral()
   google()
 
-  def found = false
-  def defaultDir = null
-  def androidSourcesName = 'React Native sources'
-
-  if (rootProject.ext.has('reactNativeAndroidRoot')) {
-    defaultDir = rootProject.ext.get('reactNativeAndroidRoot')
-  } else {
-    defaultDir = new File(
-      projectDir,
-      '/../../../node_modules/react-native/android'
-    )
-  }
+  if (REACT_NATIVE_MAJOR_VERSION == 0 && REACT_NATIVE_MINOR_VERSION < 71) {
+    def androidSourcesDir = null
+    def androidSourcesName = 'React Native sources'
 
-  if (defaultDir.exists()) {
-    maven {
-      url defaultDir.toString()
-      name androidSourcesName
+    if (rootProject.ext.has('reactNativeAndroidRoot')) {
```

Comment:
What is `reactNativeAndroidRoot`? What is it used for?

Reply:
@cortinico it would've been useful for cases such as monorepo or non-standard project structure when we couldn't locate the `react-native` folder, but actually, we don't need this here anymore since the `REACT_NATIVE_NODE_MODULES_DIR` (that I adapted from reanimated) config does the same thing. I'll remove this one.

Reply:
Got it thank you 👍 I suspected that this was the case.